RIPS/rip-7560.md

```diff
@@ -106,7 +106,8 @@ The following table represents a full list of fields of an RIP-7560 transaction:
 | paymaster                     | DATA, 20 Bytes | Address of the Paymaster contract (optional)                                        |
 | paymasterData                 | DATA           | Data that is provided to the Paymaster contract (if `paymaster` is set)             |
 | executionData                 | DATA           | Data that is provided to the Account contract for execution                         |
-| nonce                         | QUANTITY       | A 256 bit nonce. Use of `nonce > 2^64` is defined in RIP-7712                       |
+| nonceKey                      | QUANTITY       | A 192 bit nonce key. Use of `nonceKey != 0` is defined in RIP-7712.                 |
+| nonceSequence                 | QUANTITY       | A 64 bit nonce sequence. Use of `nonceKey != 0` is defined in RIP-7712.             |
```

Comment:
I think its better to stick with "nonce" here. it will complicate implementation, as it removes a standard "nonce" field which is already in use.